content/source/docs/cloud/api/audit-trails.html.md

```diff
@@ -21,6 +21,7 @@ page_title: "Audit Trails - API Docs - Terraform Cloud"
 [JSON API error object]: https://jsonapi.org/format/#error-objects
 
 # Audit Trails API
+-> **Note:** Audit Trails API is currently not available for Terraform Enterprise.
```

Comment:
Hi! Thanks for adding this :-) 

My first thought is that "currently" might imply that it will someday be supported in TFE. Even if we think it will be now, that could change so it's better to just say that it's not supported. 

I also noticed that we have four notes in a row up here at the top of this page. Is there any way that we could pull some of these notes out and put them in the text? I think we want one note and then the rest should be text - when we put everything in a separate note like this, it's hard for users to know what's important and what's not and it can be really overwhelming. What if we did something like this:

"Terraform Cloud retains 14 days of audit log information. The audit trails API exposes a stream of audit events, which describe changes to the application entities (workspaces, runs, etc.) that belong to a Terraform Cloud organization. Audit trails are a paid feature that is available in the **Terraform Cloud for Business** upgrade package. Refer to the [Terraform Cloud pricing](https://www.hashicorp.com/products/terraform/pricing) page for more details.

~> **Note:** The Audit Trails API is not available for Terraform Enterprise. 

Unlike other Terraform Cloud APIs, the audit trails API does not use the [JSON API specification](./index.html#json-api-formatting)."

^^and maybe we should also just say what kind of formatting it DOES use? or explain why it's different?

Reply:
Thanks for the comments, I have updated with your suggestions. I am not exactly sure why the audit trails API does not use the JSON API spec or how it is different compare to the other API. Someone from engineering should probably chime in here.

Reply:
@nicklhw I will reach out to you in DM about the above and hopefully the little I do know can help provide some insight.